jenkins/validate-artifacts/validate-artifacts.jenkinsfile

```diff
@@ -175,5 +226,48 @@ pipeline {
                 }
             }
         }
+        stage('Validate artifacts from urls'){
+            when {
+                expression {
+                    VERSION == ''
+                }
+            }
```

Comment:
Why do we need this check? Sometimes if a user adds both URLS and version I believe URL takes precedence? In that case add an warning statement stating `VERSION is not empty, ignoring VERSION since URLs were specified`

Reply:
I found multiple stages clumsy for version and url's validation. Just updated the Job to validate all the conditions in a single stage. Now it should be straightforward.